libs/component-library/src/lib/input/textarea.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { FxTextInput, FxTextInputProps } from './input';
+
+const FxTextArea = (props: FxTextInputProps) => (
+  <FxTextInput
+    multiline
+    height={172}
+    paddingTop="16"
+    paddingBottom="16"
```

Comment:
Could be `paddingVertical="16"`

Reply:
No it won't 😞. It's a weird bug, but `paddingVertical` doesn't actually work for multiline text inputs.